patch-db/src/store.rs

```diff
@@ -24,12 +25,46 @@ lazy_static! {
     static ref OPEN_STORES: Mutex<HashMap<PathBuf, Arc<Mutex<()>>>> = Mutex::new(HashMap::new());
 }
 
+pub struct RevisionCache {
```

Comment:
Like a buffer?

Reply:
yep, like your `RingBuffer`

Reply:
Ah but I realize what you're asking. No, we are not buffering patches to batch writes